README.md

```diff
@@ -130,6 +130,14 @@ running the connector.
 
 Optional string parameter to specify the port of the Cumulocity server. Default: "8883"
 
+#### Certificate URL (CertificateUrl)
+
+Optional string parameter to specify the URL of the Cumulocity server's certificate. 
+Default: "https://certs.godaddy.com/repository/gdroot-g2.crt"
+
+This parameter may be useful when connecting to Cumulocity Edge servers which use a self-signed
```

Comment:
Is there ever a situation where it is not useful when connecting to an edge server using a self-signed cert? If so you could remove the "may" as its very passive.

Reply:
Reworded